tests/tensor_search/test_search.py

```diff
@@ -54,6 +56,77 @@ def test_each_doc_returned_once(self):
         )
         assert len(search_res['hits']) == 2
 
+    @mock.patch.dict('os.environ', {**os.environ, **{'MARQO_MAX_SEARCHABLE_TENSOR_ATTRIBUTES': '2'}})
+    def test_search_with_excessive_searchable_attributes(self):
+        with self.assertRaises(InvalidArgError):
+            tensor_search.add_documents(
+                config=self.config, index_name=self.index_name_1, docs=[
+                    {"abc": "Exact match hehehe", "other field": "baaadd", "_id": "5678"},
+                    {"abc": "random text", "other field": "Close match hehehe", "_id": "1234"},
+                ], auto_refresh=True)
+            tensor_search.search(
+                config=self.config, index_name=self.index_name_1, text="Exact match hehehe",
+                searchable_attributes=["abc", "def", "other field"], return_doc_ids=True
+            )
+
+
+    @mock.patch.dict('os.environ', {**os.environ, **{'MARQO_MAX_SEARCHABLE_TENSOR_ATTRIBUTES': '2'}})
+    def test_search_with_allowable_num_searchable_attributes(self):
+        tensor_search.add_documents(
+            config=self.config, index_name=self.index_name_1, docs=[
+                {"abc": "Exact match hehehe", "other field": "baaadd", "_id": "5678"},
+                {"abc": "random text", "other field": "Close match hehehe", "_id": "1234"},
+            ], auto_refresh=True)
+        tensor_search.search(
+            config=self.config, index_name=self.index_name_1, text="Exact match hehehe",
+            searchable_attributes=["other field"], return_doc_ids=True
+        )
+
+    @mock.patch.dict('os.environ', {**os.environ, **{'MARQO_MAX_SEARCHABLE_TENSOR_ATTRIBUTES': '0'}})
+    def test_search_with_excessive_searchable_attributes(self):
```

Comment:
exact same name as other unit test above ^^ , please make it unique so it runs properly

Reply:
These are practically equivalent. Will remove.